Client/Frontend/Browser/BraveWebView.swift

```diff
@@ -46,4 +46,35 @@ class BraveWebView: WKWebView {
         lastHitPoint = point
         return super.hitTest(point, with: event)
     }
+
+    private func generateJavascriptFunctionString(functionName: String, args: [Any]) -> String {
+        var javascript = "\(functionName)("
+        for arg in args {
+            javascript += "'\(String(describing: arg).escapeHTML())', "
+        }
+
+        // Drop , and " "
+        javascript = javascript.dropLast(2) + ")"
+        return javascript
+    }
+
+    public func evaluateSafeJavascript(functionName: String, args: [Any], completion: @escaping ((Any?, Error?) -> Void)) {
+        let javascript = generateJavascriptFunctionString(functionName: functionName, args: args)
+        self.evaluateJavaScript(javascript) { data, error  in
+            completion(data, error)
+        }
+    }
+}
+
+extension String {
+    /// Encode HTMLStrings
+    func escapeHTML() -> String {
+       var result = self.replacingOccurrences(of: "&", with: "&amp;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: "\"", with: "&quot;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: "'", with: "&#39;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: "<", with: "&lt;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: ">", with: "&gt;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: "`", with: "&lsquo;", options: .literal, range: nil)
```

Comment:
`"/" to "&#x2F"` is also [recommended by OWASP](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#rule-1-html-encode-before-inserting-untrusted-data-into-html-element-content).

Reply:
Updated, thanks 👍

Reply:
"\" is a valid character in base64. I updated the code to replace "\" but created a `atob` wrapper to replace `&#x2F` with `/` before converting base64 to binary array. 

Let me know if you have any concerns with that approach.

Reply:
> `\` is a valid character in base64.

You mean `/` is a valid character right?

Honestly I think it's probably okay to leave that one out (and skip the extra wrapper about `atob`). I don't think you can do anything malicious with `/` if you don't have any of the other characters.

Reply:
It does highlight the fact that ideally we should be using a separate whitelist for each parameter (instead of a global blacklist), but I don't want to unnecessarily increase the scope and delay your fix.

Reply:
makes sense, I'll update.